int32[] dxl_ids
float32 goal_time_tolerance
float32[] control_points
